CommentAgent.py

```diff
@@ -360,12 +254,15 @@ def review_pr(self) -> bool:
         for _ in range(3):
             message_format = GPTsettings.MESSAGE_FORMAT
             self.GPT.add_message("user", message_format)
-            message_response = self.GPT.get_response(max_tokens=2500)
+            message_response = self.GPT.get_response(
```

Comment:
The change in the `max_tokens` default value is quite large. It's fine as long as you've checked the implications this will have on the behavior of your bot, and you're within the API's limits.